submissions/madmaxWMFU/google-popup/index.html

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>G00gle-style Popup</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+</head>
+
+<body>
+    <div class="container">
+        <div class="simpleNav">
```

Comment:
Try to use more semantic tags. 
Hint: in this case better use `<header>`

Reply:
That's a good point. Thanks!!!!!
In subsequent times, I will use navigation only this way.

Reply:
Done!